Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -96,10 +96,10 @@ WarpXParticleContainer::AllocData ()
 }
 
 void
-WarpXParticleContainer::AddNParticles (int lev,
+WarpXParticleContainer::AddNParticles (int /*lev*/,
                                        int n, const ParticleReal* x, const ParticleReal* y, const ParticleReal* z,
                                        const ParticleReal* vx, const ParticleReal* vy, const ParticleReal* vz,
-                                       int nattr, const ParticleReal* attr, int uniqueparticles, int id)
+                                       int /*nattr*/, const ParticleReal* attr, int uniqueparticles, int id)
```

Comment:
uhoh, maybe I should expand assert macros ;-)
(but why is this var otherwise unused, huh. not very useful then.)